engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -156,6 +199,19 @@ from the repository.
        stable-{{ site.docker_ee_version }}"
     ```
 
+    </div>
+    <div id="ppc64el_repo" class="tab-pane fade" markdown="1">
+
+    ```bash
+    $ sudo add-apt-repository \
+       "deb [arch=ppc64el] {{ download-url-base }} \
```

Comment:
Looks like we're missing the `<DOCKER-EE-URL>/ubuntu` in the generated documentation (for PPC and s390x);

<img width="964" alt="screen shot 2017-12-13 at 16 06 48" src="https://user-images.githubusercontent.com/1804568/33969006-dd333fa8-e01f-11e7-9fa5-c366cfffc2c0.png">
<img width="947" alt="screen shot 2017-12-13 at 16 06 39" src="https://user-images.githubusercontent.com/1804568/33969007-dd4b972e-e01f-11e7-9d3d-ecf129378886.png">
<img width="974" alt="screen shot 2017-12-13 at 16 06 29" src="https://user-images.githubusercontent.com/1804568/33969008-dd604c28-e01f-11e7-99a5-33a61a94f8a4.png">